.github/workflows/build-symengine.yml

```diff
@@ -0,0 +1,62 @@
+name: Build symengine
+
+on:
+  push:
+    branches:
+      - 'symengine-upload-new/**'
```

Comment:
do you want to have this on other branches, too?

Reply:
No, the idea is that in order to upload a new symengine version (which won't happen very often), one just has to create a branch with a name matching that pattern, modify the symengine recipe and push it.

Reply:
Sounds like a good idea! Thank you for explaining this.